src/plugins/kibana_react/public/table_list_view/table_list_view.tsx

```diff
@@ -100,34 +133,37 @@ class TableListViewUi extends React.Component {
       this.setState({
         hasInitialFetchReturned: true,
         isFetchingItems: false,
-        items: (!filter ? _.sortBy(response.hits, 'title') : response.hits),
+        items: !filter ? sortBy(response.hits, 'title') : response.hits,
         totalItems: response.total,
         showLimitError: response.total > this.props.listingLimit,
       });
     }
   }, 300);
 
   fetchItems = () => {
-    this.setState({
-      isFetchingItems: true,
-    }, this.debouncedFetch.bind(null, this.state.filter));
-  }
+    this.setState(
+      {
+        isFetchingItems: true,
+      },
+      this.debouncedFetch.bind(null, this.state.filter)
+    );
+  };
 
   deleteSelectedItems = async () => {
-    if (this.state.isDeletingItems) {
+    if (this.state.isDeletingItems || !this.props.deleteItems) {
       return;
     }
     this.setState({
-      isDeletingItems: true
+      isDeletingItems: true,
     });
     try {
-      const itemsById = _.indexBy(this.state.items, 'id');
+      const itemsById = indexBy(this.state.items, 'id');
       await this.props.deleteItems(this.state.selectedIds.map(id => itemsById[id]));
     } catch (error) {
-      toastNotifications.addDanger({
+      npStart.core.notifications.toasts.addDanger({
```

Comment:
`core.notifications` should either get passed in as a prop or be pulled out of the `useKibana()` hook. In both cases you have to make sure the places where the component is used do pass the core services in correctly. AFAIK it's used in visualize, dashboard, graph.